src/extensions/queue-repeat/queue-repeat.js

```
@@ -0,0 +1,409 @@
+    async function pollForNewQueueTracks() {
+        if (!isActive) return;
+
+        try {
+            const currentQueueUris = await getAllQueueTracks();
+            const repeatSet        = new Set(repeatList);
+            const newUris          = currentQueueUris.filter(uri => !repeatSet.has(uri));
+
+            if (newUris.length > 0) {
+                repeatList.push(...newUris);
+                log(`${newUris.length} new track(s) added to repeat list.`);
+                newUris.forEach(uri => log(`  + ${uri}`));
```

Biome (2.4.13)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)